src/components/Steps/index.tsx

```diff
@@ -1,72 +1,181 @@
+import every from 'lodash/every';
 import * as React from 'react';
-import styled from 'styled-components';
+import styled, { css } from 'styled-components';
 import asRendition from '../../asRendition';
 import { DefaultProps, RenditionSystemProps, Theme } from '../../common-types';
+import { Arrow } from '../../internal/Arrow';
 import { DismissableContainer } from '../../internal/DismissableContainer';
 import { px } from '../../utils';
 import { Flex } from '../Flex';
 import Heading from '../Heading';
 import Link from '../Link';
 import Txt from '../Txt';
-import { Arrow } from './Arrow';
 
 export interface InternalStepsProps extends DefaultProps {
+	bordered?: boolean;
+	ordered?: boolean;
+	activeStepIndex?: number;
 	titleText?: string;
 	titleIcon?: React.ReactNode;
 	onClose?: () => void;
 }
 
+type statusOptions = 'none' | 'pending' | 'completed';
+
 export interface StepProps {
 	children: string;
-	status: 'none' | 'pending' | 'completed';
+	status: statusOptions;
 	onClick?: () => void;
 }
 
-const getStatusColor = (theme: Theme, status: StepProps['status']) => {
-	switch (status) {
-		case 'pending':
-			return theme.colors.tertiary.semilight;
-		case 'completed':
-			return theme.colors.success.main;
+interface InternalStepProps extends StepProps {
+	index: number;
+	active?: boolean;
+	ordered?: boolean;
+}
+
+const HeaderContainer = styled(Flex)`
+	min-height: ${props => px(props.theme.fontSizes[5])};
+`;
+
+const getIconBg = (
+	theme: Theme,
+	ordered: boolean,
+	status: statusOptions,
+	active: boolean,
+) => {
+	let bgColor = ordered ? '#fff' : theme.colors.tertiary.semilight;
+	const completed = status === 'completed';
+	if (active) {
+		bgColor = theme.colors.primary.main;
+	} else if (completed) {
+		bgColor = theme.colors.success.main;
 	}
+	const size = px(theme.fontSizes[4]);
+	return css`
+		border-radius: 50%;
+		border: ${completed || active || !ordered
+			? 'none'
+			: `solid 1px ${theme.colors.tertiary.semilight}`};
+		color: ${!ordered || completed || active
+			? '#fff'
+			: theme.colors.tertiary.semilight};
+		background-color: ${bgColor};
+		height: ${size};
+		width: ${size};
+	`;
 };
 
-const StatusIconPlaceholder = styled.span<{ status: StepProps['status'] }>`
-	font-size: ${props => px(props.theme.fontSizes[4])};
-	line-height: ${props => px(props.theme.fontSizes[4])};
-	color: ${props => getStatusColor(props.theme, props.status)};
+const StepIconBg = styled(Flex)<{
+	ordered: boolean;
+	status: statusOptions;
+	active: boolean;
+}>`
+	${props => getIconBg(props.theme, props.ordered, props.status, props.active)};
 `;
 
-const HeaderContainer = styled(Flex)`
-	min-height: ${props => px(props.theme.fontSizes[5])};
+const StepTxt = styled(Txt)<{ active: boolean }>`
+	color: ${props => (props.active ? '' : props.theme.colors.tertiary.light)};
 `;
 
-export const Step = ({ status, children, onClick }: StepProps) => {
+const StepLink = styled(Link)`
```

Comment:
Why should this be overridden? IMO all links should look the same, and if we think the current one doesn't look as expected, we need to update the Link component itself.